src/Services/Catalog/Catalog.API/Extensions/RedisCacheExtensions.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.IO;
+
+using System.Threading.Tasks;
+using System.Runtime.Serialization.Formatters.Binary;
+
+using Microsoft.Extensions.Caching.Distributed;
+using System.Collections.Generic;
+
+namespace Catalog.API.Extensions
+{
+    public static class RedisCacheExtensions
+    {
+        public static readonly string KEYS = "KEYS";
```

Comment:
We should adhere to naming convention.

Reply:
How about changing from KEYS to RedisKeys?

Reply:
Sounds good.